extra/Dockerfile.armv7-armhf

```diff
@@ -50,7 +54,11 @@ RUN apt-get update && \
     git \
     python-pip \
     python-setuptools \
-    python-wheel && \
+    python-wheel \
```

Comment:
Please keep the contents in Docker images minimal. Docker images are meant for runtime not for development. Feel free to install additional tools, when starting your container.

Reply:
The apt packages which I added to the arm Dockerfile are actually needed to successfully compile lxml. The ubuntu Dockerfile seems to not have this issue.

Reply:
All of the added dependencies are only needed for compiling and are uninstalled afterwards (see further in the Dockerfile). Except libxslt1.1 . It must be kept, otherwise the program crashes.

Reply:
I now removed all the added apt dependencies into a pre staging docker container.

Reply:
Also i removed zlib1g-dev which was not needed anymore already since you changed to install pillow from apt instead of pip.